ui/src/app/pages/admin/user/admin-user-bypass.tsx

```diff
@@ -232,6 +238,34 @@ export class AdminUserBypass extends React.Component<Props, State> {
                 }}
               />
             )}
+            <Toggle
+              name='Profile Confirmation'
+              checked={selectedModules.includes(
+                AccessModule.PROFILECONFIRMATION
+              )}
+              data-test-id='profile-confirmation-toggle'
+              onToggle={() => {
+                this.setState({
+                  selectedModules: fp.xor(selectedModules, [
+                    AccessModule.PROFILECONFIRMATION,
+                  ]),
+                });
+              }}
+            />
+            <Toggle
+              name='Publication Confirmation'
+              checked={selectedModules.includes(
+                AccessModule.PUBLICATIONCONFIRMATION
+              )}
+              data-test-id='publication-confirmation-toggle'
+              onToggle={() => {
+                this.setState({
+                  selectedModules: fp.xor(selectedModules, [
+                    AccessModule.PUBLICATIONCONFIRMATION,
+                  ]),
+                });
+              }}
+            />
```

Comment:
Consider something like:
```javascript
<FlexColumn style={{ padding: '1.5rem' }}>
<Fragment>
[
  enableComplianceTraining && {
    moduleName: AccessModule.COMPLIANCETRAINING,
    name: 'RT Compliance Training',
    'data-test-id': 'rt-compliance-training-toggle'
  },
  ...
].filter(x => x) // remove falsy values
.map(m =>
  <Toggle
    name={m.name}
    checked={selectedModules.includes(m.module)}
    data-test-id={m['data-test-id']}
    onToggle={() => {
      this.setState({selectedModules: fp.xor(selectedModules, [m.module])})
    }}
  />
)
</Fragment>
...
```
This could potentially allow you to consolidate some tests, since it would make it clear that the implementation does not change between modules.

Reply:
Good suggestion. I'll do this in one of the follow up refactor PRs.